client/platform/desktop/frontend/components/Recent.vue

```diff
@@ -100,16 +104,17 @@ export default defineComponent({
     async function finalizeImport(args: DesktopMediaImportResponse) {
       importing.value = true;
       await request(async () => {
-        const jsonMeta = await api.finalizeImport(args);
+        const conversionArgs = await api.finalizeImport(args);
+        api.queueConversion(conversionArgs);
         pendingImportPayload.value = null; // close dialog
-        if (!jsonMeta.transcodingJobKey) {
+        if (!conversionArgs.mediaList || conversionArgs.mediaList.length === 0) {
           router.push({
             name: 'viewer',
-            params: { id: jsonMeta.id },
+            params: { id: conversionArgs.meta.id },
           });
         } else {
           // Display new data and await transcoding to complete
-          const recentsMeta = await api.loadMetadata(jsonMeta.id);
+          const recentsMeta = await api.loadMetadata(conversionArgs.meta.id);
```

Comment:
We need some way to either prevent a user from clicking on a queued for transition item in the recents.  Maybe we check the queued list so if it isn't done we add some sort of label (v-chip) or icon and make the clicking on it disabled.  When converting you can't click on it, so when queued you shouldn't be able to open it either.

Reply:
Done in 795d988